openedx/core/djangoapps/site_configuration/models.py

```diff
@@ -89,15 +89,27 @@ def get_all_orgs(cls):
         for example, to do filtering.
 
         Returns:
-            Configuration value for the given key.
+            A list of all organizations present in site configuration.
         """
         org_filter_set = set()
-        for configuration in cls.objects.filter(values__contains='course_org_filter', enabled=True).all():
+
+        configurations = cls.objects.filter(values__contains='course_org_filter', enabled=True).all()
+        for configuration in configurations:
             org_filter = configuration.get_value('course_org_filter', None)
             if org_filter:
                 org_filter_set.add(org_filter)
         return org_filter_set
```

Comment:
At some point we should probably, move the site/organization association out of the JSON configuration field to a many-to-many relationship. As things stand now, it is not possible to associate more than one organization with a Site. You could do this with microsite configuration (although we have not utilized that ability for WL sites as of yet).